src/libs/ajax/leonardo/models/core-models.ts

```diff
@@ -0,0 +1,27 @@
+import { CloudProvider } from 'src/libs/workspace-utils'
+import { ToolLabel } from 'src/pages/workspaces/workspace/analysis/tool-utils'
+
+
+export interface AuditInfo {
+  creator: string
+  createdDate: string
+  destroyedDate?: string
+  dateAccessed: string
+}
+
+export interface BaseLabels {
+  tool: ToolLabel
+}
+
+export type LeoResourceLabels = BaseLabels & Record<string, any>
+
+export interface CloudContext {
+  cloudProvider: CloudProvider
```

Comment:
CloudPlatform is what the UIWG agreed will be what we use to talk about Azure vs GCP.
The primary reason being "provider" is a keyword for react state management.

Reply:
This is essentially the decoder for a leo API response, so the name cannot be changed. The type can to conform, but I believe that is out-of-scope for this ticket's goals. What is the ticket number for your platform/provider consistency updates?

Reply:
https://broadworkbench.atlassian.net/browse/IA-3971

It's currently just a placeholder ticket.
Ah so this is one of the spots where we can't change it?